sdk/src/builder.rs

```diff
@@ -1422,6 +1418,77 @@ impl Builder {
         Ok(self)
     }
 
+    #[async_generic(async_signature(
+        &mut self,
+        time_authority_url: &str,
+        store: &mut Store,
+        http_resolver: &impl AsyncHttpResolver,
+    ))]
+    fn maybe_add_timestamp(
+        &mut self,
+        time_authority_url: &str,
+        store: &mut Store,
+        http_resolver: &impl SyncHttpResolver,
+    ) -> Result<()> {
+        if self.settings.builder.add_timestamp_assertion_to_parent {
```

Comment:
This should be simplified to just return when add_timestamp_assertion_to_parent is false.  Per our code guideline we should favor that instead of a whole function if statement